### ISSUE TITLE
Feature Request: Tiered NPC Tags should update based on tier

### ISSUE THREAD

**Issue by compcon** [massif-press/compcon#573]
<strong>Submitted At: </strong>2/2/2020 12:21:40
<strong>Platform: </strong>
Browser - Chrome
<strong>Describe the bug: </strong>
Reliable does not respond to tier 
<strong>To Reproduce: </strong>
Add any NPC weapon with reliable
<strong>Expected behavior: </strong>
reliable should scale with tier like everything else
<strong>Additional context: </strong>

<strong>Contact Info: </strong>
Nice#0367

**Comment by byronhulcher** [comment]
I looked into this briefly this morning.  According to the GM Section of the Lancer Core Book the following NPCs have reliable:

Archer - Light Machine Gun - Reliable 3/4/5
Assault - Heavy Assault Rifle - Reliable 2/3/4
**Rainmaker - Missile Pods - Reliable 2/3/3**

While I don't have access to the GM JSON data, I did check in COMPCON and validated in the NPC Roster, Encounter Builder, and Mission Runner that these stats were accurately reflected for all 3 NPC classes.  I believe that reporter might have been confused and assumed that the Rainmaker's `Reliable 2/3/3` was incorrect data when it appears in the rules as written.

**Comment by aritsune** [comment]
nah, I don't think this is that this report is getting at. I think they want the Reliable tag to only show the value of the relevant tier, which is technically more of a feature request, but I can't blame someone unfamiliar with the code for filing it as a bug
